package.json

```diff
@@ -12,11 +12,14 @@
   ],
   "scripts": {
     "build": "webpack -p",
+    "start": "webpack-dev-server --config ./webpack.dev.js",
     "lint": "eslint . --ext .jsx --ext .js",
     "pretest": "npm run lint && npm run prettier",
     "prettier": "prettier --list-different --write \"./**/**.{js,jsx}\"",
-    "test": "jest --coverage --runInBand",
+    "test": "jest --coverage --runInBand --testPathIgnorePatterns '__tests__/browser'",
```

Comment:
@rafegoldberg how does this make you feel?

The way it's setup you should be able to run `npx jest path/to/any/file` and it works. The downside is that the package script has to know the paths. 

`jest` has the concept of `packages`, but they're path based. So you could do something like:

```
"test.browser": "jest --projects __test__/browser",
```

and put a `jest.config.js` in `__test__/browser`. But, the config gets annoying because `jest`'s `<rootDir>` gets set to that dir. It just seems easier to keep them more similar.

Reply:
Thinking about this more today; let's chat "offline" today about the scripts/file structure/naming conventions in this PR?